### PR TITLE
Image Classification on Flink streaming tutorial

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: docs/docs/tutorials/image-classification-Flink.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+- [Running the Flink program on a local machine or a cluster](#running-the-flink-program-on-a-local-machine-or-a-cluster)
+  - [Configuring and starting Flink](#Configuring-and-starting-Flink)
+  - [Building the project](#Building-the-project)
+  - [Running the example](#Running-the-example)
+
+## Getting started this tutorial
+
+### Summary
+
+This is the real-time image classification on Apache Flink streaming. Images extracted from ImageNet database will be predicted with pre-trained MobileNet_v1 model which is loaded by Analytics Zoo Inference model.
```
Comment (glorysdj) [inline docs/docs/tutorials/image-classification-Flink.md:27]: by Analytics Zoo InferenceModel？

File: docs/docs/tutorials/image-classification-Flink.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+Apache Flink is a powerful stream processing framework that supports batch processing, data streaming programs and running in common cluster environment. 
+
+Let's get started this tutorial. We will use an example to introduce how to load pre-trained tensorflow model as Analytics Zoo Inference Model TFNet for the real-time image prediction on Flink streaming. See the whole program [here](https://github.com/intel-analytics/analytics-zoo/tree/master/apps/model-inference-examples/model-inference-flink/src/main/scala/com/intel/analytics/zoo/apps/model/inference/flink/ImageClassification). 
+
+### Prerequisites
+
+- **Environment preparation**
+
+Make sure JDK 1.8, Scala 2.11/2.12, Flink 1.8.1, and Maven are installed.
```
Comment (glorysdj) [inline docs/docs/tutorials/image-classification-Flink.md:43]: do we need scala installed?

Comment (Le-Zheng) [reply docs/docs/tutorials/image-classification-Flink.md]: Ok, we really don't need it.

File: docs/docs/tutorials/image-classification-Flink.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+#### Knowing about dataset
+
+The raw images in **ImageNet** are various sizes.  Let us show two of the predicting images.
+
+<img src="../../../zoo/src/test/resources/imagenet/n02110063/n02110063_15462.JPEG" width="180">       
+<img src="../../../zoo/src/test/resources/imagenet/n04370456/n04370456_5753.JPEG" width="180">
+
+#### Preprocessing dataset
+
+MobileNet_v1_1.0_224 network has the image input of 224 by 224. Inference Model requires predict Input is  `JList[JList[JTensor]]`.
```
Comment (qiyuangong) [inline docs/docs/tutorials/image-classification-Flink.md:82]: MobileNet_v1_1.0_224's input layer is 224 * 224 * 3.

Inference Model requires predict Input to be ...

File: docs/docs/tutorials/image-classification-Flink.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+The raw images in **ImageNet** are various sizes.  Let us show two of the predicting images.
+
+<img src="../../../zoo/src/test/resources/imagenet/n02110063/n02110063_15462.JPEG" width="180">       
+<img src="../../../zoo/src/test/resources/imagenet/n04370456/n04370456_5753.JPEG" width="180">
+
+#### Preprocessing dataset
+
+MobileNet_v1_1.0_224 network has the image input of 224 by 224. Inference Model requires predict Input is  `JList[JList[JTensor]]`. 
+
+In this example, `trait ImageProcessing` is prepared to provide approaches to convert format, resize and normalize. These methods are defined [here](https://github.com/intel-analytics/analytics-zoo/blob/master/apps/model-inference-examples/model-inference-flink/src/main/scala/com/intel/analytics/zoo/apps/model/inference/flink/ImageClassification/ImageProcessing.scala).
```
Comment (qiyuangong) [inline docs/docs/tutorials/image-classification-Flink.md:84]: trait?

File: docs/docs/tutorials/image-classification-Flink.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+```
+Siberian husky
+sweatshirt
+```
+
+#### Wrapping up
+
+We have reached the end of the tutorial. In this tutorial, we introduce how to use Analytics Zoo Inference Model for image classification on streaming. We write a subclass extends `InferenceModel`, and load pre-trained model. With that, we define  `RichMapFunction` and started with the prediction on Flink streaming.
+
+What goes for next? You could take practice. Load the data and pre-trained model you need to see what speedup you get.
```
Comment (qiyuangong) [inline docs/docs/tutorials/image-classification-Flink.md:318]: how much speedup?

**Pull request merged**